geomesa-lambda/geomesa-lambda-tools/bin/install-kafka.sh

```diff
@@ -16,6 +16,7 @@ declare -a urls=(
   "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka-clients/$kafka_version/kafka-clients-$kafka_version.jar"
   "https://search.maven.org/remotecontent?filepath=org/apache/zookeeper/zookeeper/$zookeeper_version/zookeeper-$zookeeper_version.jar"
   "https://search.maven.org/remotecontent?filepath=com/101tec/zkclient/$zkclient_version/zkclient-$zkclient_version.jar"
+  "https://search.maven.org/remotecontent?filepath=net/sf/jopt-simple/jopt-simple/%%jopt-simple.version%%/jopt-simple-%%jopt-simple.version%%.jar"
```

Comment:
i think so... i did some testing with the script and some with KAFKA_HOME so can't verify 100%